### PR TITLE
Alerts. Toast and Snackbar

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**

### CODE REVIEW

**Comment by PRMerger16** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/TOC.yml [hunk 1]
```diff
@@ -5,6 +5,14 @@ items:
...
     - name: Get started
       href: get-started.md
+- name: "Alerts"
+  items:   
+    - name: Overview
+      href: alerts/index.md
+    - name: "Toast"
+      href: alerts/toast.md
+    - name: "Snackbar"
+      href: alerts/snackbar.md
```
Comment (bijington) [inline docs/maui/TOC.yml:15]: Can we order this alphabetically?

```suggestion
- name: "Alerts"
  items:   
    - name: Overview
      href: alerts/index.md
    - name: "Snackbar"
      href: alerts/snackbar.md
    - name: "Toast"
      href: alerts/toast.md
```

File: docs/maui/alerts/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# Alerts
+
+## .NET MAUI Community Toolkit Alerts
+
+The .NET MAUI Community Toolkit extends the list of .NET MAUI alerts. Here are the alerts provided by the toolkit:
+
+| Alert | Description |
+| --------- | ----------- |
+| [`Toast`](toast.md) | The `Toast` is a timed alert that appears at the bottom of the screen by default. It is dismissed after a configurable duration of time. |
+| [`Snackbar`](snackbar.md) | The `Snackbar` is a timed alert that appears at the bottom of the screen by default. It is dismissed after a configurable duration of time. `Snackbar` is fully customizable and can be anchored to any `IView`. |
```
Comment (bijington) [inline docs/maui/alerts/index.md:17]: Can we order this alphabetically?
```suggestion
| [`Snackbar`](snackbar.md) | The `Snackbar` is a timed alert that appears at the bottom of the screen by default. It is dismissed after a configurable duration of time. `Snackbar` is fully customizable and can be anchored to any `IView`. |
| [`Toast`](toast.md) | The `Toast` is a timed alert that appears at the bottom of the screen by default. It is dismissed after a configurable duration of time. |
```

**Review by bijington — COMMENTED** [review]
Thanks @VladislavAntonyuk I have just added some minor suggestions but it looks good.

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by brminnick — APPROVED** [review]
Thanks Vlad!!

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Pull request merged**